ui/app/ducks/gas/gas.duck.js

```diff
@@ -451,6 +463,52 @@ export function fetchGasEstimates (blockTime) {
   }
 }
 
+export function fetchMetaSwapsGasPriceEstimates () {
```

Comment:
This seems like a bad idea. The `gas` slice will now contain basic gas estimates from either ETH Gas Station or from the swaps API, depending on which of these was called last.

They're being cleared when the swaps route is entered or left I know, but why are they sharing space in the first place? It's bad enough that we have this area of global state being used for different purposes already (in confirmations and the send flow) - lets not add a third place, and further complicate the one simple piece of data in this slice (the basic gas estimates).

Reply:
This issue went away with the refactor (d0aaf76e1 and 9de36610a)